src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,104 @@ This new object is an array of example objects that can successfully POST agains
 Updating an existing schema
 ----------------
 
-There are 2 categories when we talk about updating an existing schema:
+There are two situations we need to consider when updating an existing schema:
 
-* No affect on schema version
-* Update schema version
+### No change on schema version
 
+* For some schema changes, there will be no change to the schema version. These will be some minor changes to the schema. One such example includes adding a new enum to an existing list of enums. Another example will be adding a new property to existing schemas.
 
-**The schema version is updated, requiring an upgrade of the schema, when the objects stored in the database will not validate and POST to the updated schema.**
+**Follow the steps as outlined below**
 
-
-### No affect on schema version
-
-1. In the **schemas** directory, make edits  to properties in the JSON file named after the object.
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object.
 
 2. In the **types** directory, make appropriate updates to object class by adding *embedding*, *reverse links*, and *calculated properties* as necessary.
 
 3. Update sample data, **data/inserts** directory, to test the changes made to the schema in **tests** directory. 
```

Comment:
You need to include the change log update requirement and provide example of "minor change since schema ..."